src/domManipulation.ts

```diff
@@ -249,7 +324,7 @@ export function closePopup() {
 }
 
 // Controls what happens when user clicks on the map to select own location.
-export function handleMapClick(mapClickEvent) {
+export function handleMapClick(mapClickEvent: { latlng: { wrap: () => any; }; }) {
```

Comment:
Please don't use `any` try instead `function onMapClick(e: { latlng: { lat: number; lng: number } })`

Reply:
without the `e` that is if you used an event

Reply:
Use the LatLng type from leaflet:
```ts
import { LatLng } from 'leaflet';

export function handleMapClick(mapClickEvent: { latlng:  LatLng }) { ... }

```

Reply:
I have updated my implementation to match these suggestions.